### ISSUE TITLE
Mail settings do not auto reconnect after mail server reboot.

### ISSUE THREAD

**Issue by MrDecepticon** [freescout-helpdesk/freescout#2364]
Was performing mail server updates last night.  When all was finished and rebooted, Freescout doesn't seem to send email notifications to help desk users.  It appears to fetch emails from the support mailbox that is setup but I have to go into the mail settings and run a test connection and save (without changing any settings) and then it will start sending emails out.  It seems to affect client reply emails as well. 

From the log but this was expected as that is the time that the mail server rebooted:

<html>
<body>

Nov 2, 09:48pm:05 | Error fetching email | Error: imap_open(): Couldn't open stream {mail.lobar.com:993/imap/ssl}. Can't connect to mail.lobar.com,993: No route to host; File: /var/www/html/overrides/webklex/laravel-imap/src/IMAP/Client.php (230
-- | -- | --

</body>
</html>

No other errors after that.

**Comment by MrDecepticon** [comment]
THere are three more log entries with that error.  Is there a threshold of trying to connect before Freescout disables the connection?

**Comment by freescout-helpdesk** [comment]
Yes. There is some timeout, but in your case the problem is not in timeout but in `No route to host`. Nothing can be done on FreeScout side in this case.

**Comment by MrDecepticon** [comment]
Not sure why this was closed.  Maybe I phrased it incorrectly?  

The no route to host was expected when the mail server was rebooting. 

> From the log but this was expected as that is the time that the mail server rebooted:

 My issue is, Freescout never started checking/fetching emails OR sending emails for at least 6 hours AFTER the mail server reboot was complete and mail was flowing again.  Freescout only started pushing email/notifications after I 'tested and saved' the email settings.  At that point, Freescout fetched all the email and pushed email out all at once. 

Is there a threshold of how many times freescout will try to contact the mail server before it gives up?  If so is there a way to increase that time/attempts or disable it all together?  Otherwise, do I need to always go in and re-save the mail server settings after a mail server reboot in order for freescout to start fetching again?
